google/cloud/storagecontrol/v2/.repo-metadata.json

```diff
@@ -6,7 +6,7 @@
     "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:187243%20status=open",
     "language": "cpp",
     "library_type": "GAPIC_AUTO",
-    "name_pretty": "Cloud Storage API",
+    "name_pretty": "Storage Control API",
```

Comment:
Sounds like we may want to change some stuff in the README file.  I will send a PR if needed.

Reply:
> I will send a PR if needed.

I see you sent a PR.